recipes/gdal2/bld.bat

```diff
@@ -0,0 +1,58 @@
+
+REM echo %PREFIX%
+REM echo %LIBRARY_INC%
+REM echo %LIBRARY_LIB%
+REM exit 1
```

Comment:
No need for those here. Since this will be a community recipe we try to keep them as clean as possible for the reviewers.